templates/arm-vnet-template.json

```diff
@@ -438,6 +442,21 @@
         "[resourceId('Microsoft.Network/networkSecurityGroups',parameters('appgwNsgName'))]",
         "[resourceId('Microsoft.Network/networkSecurityGroups',parameters('iaasNsgName'))]"
       ]
+    },
+    {
+      "type": "Microsoft.Network/privateDnsZones/virtualNetworkLinks",
+      "apiVersion": "2018-09-01",
+      "name": "[concat(variables('privateDnsZone')[copyIndex()].name, '/', parameters('aksVnetName') )]",
+      "location": "global",
+      "dependsOn": [
+          "[resourceId('Microsoft.Network/privateDnsZones', variables('privateDnsZone')[copyIndex()].name)]"
```

Comment:
don't you need to pass the resource group as well? if it's not in the same resource group?

Reply:
Good point but it doesn't look like I need that, judging by the exported template I am looking at.